src/pybuffer.jl

```diff
@@ -86,14 +93,16 @@ function Base.stride(b::PyBuffer, d::Integer)
     return Int(unsafe_load(b.buf.strides, d))
```

Comment:
all of these `unsafe_load` calls need rooting… but that is not the fault of this PR